setup.cfg

```diff
@@ -63,7 +63,13 @@ include = torchgeo*
 # Optional dataset requirements
 datasets =
     h5py
+    # loading .las point clouds (idtrees) laspy 2+ required for Python 3.6+ support
+    laspy>=2.0.0
+    # open3d will add add support for python 3.9 in pypi in v0.14. v0.11.2 last version for tests to pass
```

Comment:
What in 0.11.2 is required for the tests to pass?

Reply:
I had this issue https://github.com/isl-org/Open3D/issues/1550. They recommend to install from latest so I'm not going to debug it in depth.

Reply:
Can you add a link to this issue?

Reply:
Yep will do